app/controllers/triage/WhatRoleController.scala

```diff
@@ -66,7 +65,7 @@ class WhatRoleController @Inject()(override val messagesApi: MessagesApi,
           Future.successful(BadRequest(view(formWithErrors))),
         value => {
           val uaUpdated = UserAnswers().set(WhatRoleId)(value).asOpt.getOrElse(UserAnswers())
-          Future.successful(Redirect(navigator.nextPage(WhatRoleId, NormalMode, uaUpdated)))
+          Future.successful(Redirect(controllers.triage.routes.WhatRoleController.onPageLoad()))
```

Comment:
What is the reason for removing the navigator here?

Reply:
@Ashwani1403

Reply:
The reason is earlier there were two different triage pages journey on the toggle basis. now one becomes redundant. However we need the first page redirection to newV2 triage pages on submit. Hence redirected to onPageLoad() method as it was taking to us to new triageV2 pages navigation and also avoided major changes on unit test's.